### DISCUSSION TITLE
./docker.exe not recognized (4.1.1 Windows)

### DISCUSSION THREAD

**Discussion by badekappe** [containers/podman#14749] [Q&A]
As a newbie, I followed the tutorial on https://github.com/containers/podman/blob/main/docs/tutorials/podman-for-windows.md. All went fine, untill "Using API Forwarding". The command **.\docker.exe run -it fedora echo "Hello Podman!"** results in an error:

```
.\docker.exe : The term '.\docker.exe' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ .\docker.exe run -it fedora echo "Hello Podman!"
+ ~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (.\docker.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

```

Retried the installation on a fresh machine (no previous install of WSL, Podman or Docker) with the same result.

Am I missing anything?

**Comment by Luap99** [comment]
You would need to install the docker client in order to use it. If you do not need docker just skip that.
